source/general/downloads.rst

```diff
@@ -0,0 +1,24 @@
+===================
+Available Downloads
+===================
+
+The ProjectKorra team has several projects, these are the ones available to download.
+
+Spigot
+======
+- `ProjectKorra Core plugin`_ (v1.8.8 for MC 1.13.2)
+
+Addons and Side Plugins
+-----------------------
+There are tons of addons and side plugins that community members have made, all the publicly released ones can be found `on our website`_!
+
+.. note:: The ProjectKorra team cannot provide help for unofficial resources, but can assist in finding who to contact if a problem arises.
+
+Forge
+=====
+- `Avatar Mod 2`_: Out of the Iceberg
```

Comment:
I feel there needs to be something else here. Anyone know the forge version for the mod?

Reply:
i suppose I can find out but I disagree that something else is needed

Reply:
> i suppose I can find out but I disagree that something else is needed

I think it helps consistency because that's something that people who use mods need to be aware of, just like someone who is using the plugin needs to know the compatible version.